examples/input_chicane.in

```diff
@@ -0,0 +1,64 @@
+###############################################################################
+# Particle Beam(s)
+###############################################################################
+beam.npart = 10000
+beam.units = static
+beam.energy = 5.0e3
+beam.charge = 0.0
+beam.particle = electron
+beam.distribution = waterbag
+beam.sigmaX = 2.2951017632e-5
+beam.sigmaY = 1.3084093142e-5
+#beam.sigmaT = 5.5555553e-8
```

Comment:
Maybe just leftover:
```suggestion
```

Reply:
I tried two sets of parameters for the longitudinal phase space.  Let me verify which is best, and I'll delete the second and push the change.